backend/devices/bitbox02/keystore.go

```diff
@@ -66,7 +68,49 @@ func (keystore *keystore) OutputAddress(
 		msgCoinMap[coin.Code()], configuration.AbsoluteKeypath().ToUInt32(),
 		messages.BTCPubRequest_ADDRESS, msgScriptType, true)
 	return err
+}
+
+// HasSecureBTCPubOutput implements keystore.Keystore.
+func (keystore *keystore) HasSecureBTCPubOutput() bool {
+	return true
+}
 
+func (keystore *keystore) VerifyBTCPub(coin coinpkg.Coin, keyPath signing.AbsoluteKeypath, configuration *signing.Configuration) error {
+	if !keystore.HasSecureBTCPubOutput() {
+		panic("HasSecureOutput must be true")
+	}
+	msgCoin, ok := msgCoinMap[coin.Code()]
+	if !ok {
+		return errp.New("unsupported coin")
+	}
+	var msgOutputType messages.BTCPubRequest_OutputType
+	switch specificCoin := coin.(type) {
```

Comment:
with only one `case`, you can save some indentation with:

`btcCoin, ok := coin.(*btc.Coin)`